lib/data/model.ts

```diff
@@ -283,8 +324,8 @@ export default class Model extends DenaliObject {
    * attributes
    */
   inspect(): string {
-    let attributesSummary: string[] = this.mapAttributes((value, attributeName) => {
-      return `${ attributeName }=${ JSON.stringify(value) }`;
+    let attributesSummary: string[] = (<typeof Model>this.constructor).mapAttributes(this.container, (value, attributeName) => {
+      return `${ attributeName }=${ JSON.stringify(this[attributeName]) }`;
```

Comment:
see above. When they're made static, the value itself is actually the attribute descriptor, so we need to use `this[attributeName]` to get the actual instance value